awesome_nested_set.gemspec

```diff
@@ -18,7 +18,7 @@ Gem::Specification.new do |s|
 
   s.required_ruby_version = '>= 2.0.0'
 
-  s.add_runtime_dependency 'activerecord', '>= 4.0.0', '< 7.2'
+  s.add_runtime_dependency 'activerecord', '>= 4.0.0', '< 8.0'
```

Comment:
Why 8.0 instead of 7.3?

Reply:
because the next release will be 8.0 as I know

Reply:
It's good to add the tests configuration for Rails 7.2 like it's done for 7.1 in https://github.com/collectiveidea/awesome_nested_set/commit/034ab03ce97e5f078581cffb70e2385a06415e8a

Reply:
Ok, will do that

Reply:
@ngrebenshikov updated!